test/integration/vhds_integration_test.cc

```diff
@@ -72,6 +72,16 @@ const char Config[] = R"EOF(
                     cluster_name: xds_cluster
 )EOF";
 
+const char RdsWithoutVhdsConfig[] = R"EOF(
```

Comment:
We prefer these configs to are created using the protobuf objects directly rather than parsing configs.

Reply:
This is the first time I hear about this preference. This test suit been around for a while now, with this exact setup (and it's not the only test suite to use yaml for setup). 

FWIW, configurations used in these tests have enough nested fields/protobufs that building them programmatically will be laborious and harder to read.

Reply:
I disagree re: adding vhds to one of default configs (shared test configuration couples independent tests and can make them brittle), but I see value in adding helper methods to generate VHDS config. Will, but in a separate PR if that's ok.